dart/test/mocks/mock_transport.dart

```diff
@@ -31,6 +46,10 @@ class MockTransport implements Transport {
     final envelopeMap = envelopeItemJson as Map<String, dynamic>;
     final requestJson = envelopeMap['request'] as Map<String, dynamic>?;
 
+    // TODO the following code should really be part of fromJson() that handle those keys.
+    // JSON being Map<String, dynamic> is nothing out of ordinary.
+    // See [SentryResponse.fromJson()] as an example.
```

Comment:
Is this still necessary? We already do `Map.from(...)` in the ctor.

Reply:
The TODO is here for the remainder of the code in the function, i.e. the old code not touched by this PR. I mean I could just do Map.from() for the other two affected places - Request[other] & Request[headers]

Reply:
done